src/NuGet/Microsoft.NETCore.Compilers/Microsoft.NETCore.Compilers.Package.csproj

```diff
@@ -0,0 +1,78 @@
+﻿<!-- Copyright (c)  Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information. -->
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
+
+    <IsPackable>true</IsPackable>
+    <PackageId>Microsoft.NETCore.Compilers</PackageId>
+    <IncludeBuildOutput>false</IncludeBuildOutput>
+    <DevelopmentDependency>true</DevelopmentDependency>
+    <PackageDescription>
+      CoreCLR-compatible versions of the C# and VB compilers for use in MSBuild.
+    </PackageDescription>
+    <TargetsForTfmSpecificContentInPackage>$(TargetsForTfmSpecificContentInPackage);_GetFilesToPackage</TargetsForTfmSpecificContentInPackage>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\..\Compilers\CSharp\csc\csc.csproj" PrivateAssets="all"/>
+    <ProjectReference Include="..\..\Compilers\VisualBasic\vbc\vbc.csproj" PrivateAssets="all"/>
+    <ProjectReference Include="..\..\Interactive\csi\csi.csproj" PrivateAssets="all"/>
+    <ProjectReference Include="..\..\Compilers\Core\MSBuildTask\Microsoft.Build.Tasks.CodeAnalysis.csproj" PrivateAssets="all"/>
+    <ProjectReference Include="..\..\Compilers\Server\VBCSCompiler\VBCSCompiler.csproj" PrivateAssets="all"/>
+  </ItemGroup>
+
+  <ItemGroup>
+    <_ExplicitProjectReference Include="@(ProjectReference)" />
+  </ItemGroup>
+
+  <!-- Publish dependent projects before packaging. -->
+  <Target Name="_PublishDependencies" BeforeTargets="Pack">
+    <MSBuild Projects="@(_ExplicitProjectReference)" Targets="Publish" Properties="TargetFramework=$(TargetFramework)" BuildInParallel="true" />
+  </Target>
+
+  <Target Name="_GetFilesToPackage">
+    <PropertyGroup>
+      <_Dlls>$(ArtifactsConfigurationDir)Dlls\</_Dlls>
+      <_Exes>$(ArtifactsConfigurationDir)Exes\</_Exes>
+    </PropertyGroup>
+    <ItemGroup>
+      <_File Include="$(MSBuildProjectDirectory)\build\**\*.*" TargetDir="build" />
+      <_File Include="$(MSBuildProjectDirectory)\tools\**\*.*" TargetDir="tools" />
+
+      <!-- Targets and task files -->
+      <File Include="$(_Dlls)Microsoft.Build.Tasks.CodeAnalysis\netcoreapp2.0\Microsoft.Build.Tasks.CodeAnalysis.dll" TargetDir="tools" />
```

Comment:
❓ Why are some `<File>` and some `<_File>`?

Reply:
Typo. Fixing.